packages/server/src/arpa_reporter/lib/audit-report.js

```diff
@@ -525,7 +524,7 @@ async function getCache(periodId, domain, tenantId, force = false, logger = log)
     return data;
 }
 
-async function generateSheets(periodId, domain, tenantId, dataBefore = null, logger = log) {
+async function generateSheets(periodId, domain, tenantId = useTenantId(), dataBefore = null, logger = log) {
```

Comment:
Just curious.  Any reason you chose to introduce this default this early in the stack?

Reply:
It had to do with a test case failing. Not a great reason, but I didn't touch the test case, and I try to not change test cases to make my code work.